### PR TITLE
[MRG] Fix minimizers

### PR DESCRIPTION
**Pull request by hannahbrucemacdonald**
This PR removes any use of the FIREMinimizationIntegrator, and instead uses functions that rely on openmm.LocalEnergyMinimizer instead, which has been more robust - particularly with hybrid systems.

### CODE REVIEW

**Comment by hannahbrucemacdonald** [comment]
This addresses #588

File: perses/annihilation/relative.py [hunk 1]
```diff
@@ -135,8 +135,15 @@ def __init__(self,
         self._interpolate_14s = interpolate_old_and_new_14s
 
         #new attributes from the modified geometry engine
-        self.neglected_new_angle_terms = neglected_new_angle_terms
-        self.neglected_old_angle_terms = neglected_old_angle_terms
+        if neglected_old_angle_terms:
```
Comment (jchodera) [inline perses/annihilation/relative.py]: Has this PR accidentally pulled in additional changes?

File: perses/app/setup_relative_calculation.py [hunk 2]
```diff
@@ -328,7 +324,7 @@ def run_setup(setup_options):
         _logger.info(f"\tatom selection detected: {atom_selection}")
     else:
         _logger.info(f"\tno atom selection detected: default to None.")
-        atom_selection = None
+        atom_selection = 'all'
```
Comment (jchodera) [inline perses/app/setup_relative_calculation.py]: Looks like this has changed for some reason?

File: perses/app/setup_relative_calculation.py [hunk 3]
```diff
@@ -538,10 +534,6 @@ def run_setup(setup_options):
             _logger.info(f'\tRunning {phase} phase...')
             hss_run = hss[phase]
 
-            _logger.info(f"\t\tminimizing...\n\n")
-            hss_run.minimize()
```
Comment (jchodera) [inline perses/app/setup_relative_calculation.py]: Do we really want to eliminate minimization? I think we just want to switch minimizers.

**Comment by jchodera** [comment]
Looks like there may be a bunch of unrelated changes here?

Is there a simple unit test that, for example, tests the setup/minimization process in vacuum so it can run on travis?

File: perses/annihilation/relative.py (hunk 1 above)
Comment (dominicrufa) [reply perses/annihilation/relative.py]: this is an additional change independent of the minimization issue; there was a discrepancy between `self.neglect_`... being an empty list versus a `NoneType`

File: perses/app/setup_relative_calculation.py (hunk 3 above)
Comment (hannahbrucemacdonald) [reply perses/app/setup_relative_calculation.py]: We are still minimizing, it's just done as part of the setup. We were minimizing twice previously

File: perses/app/setup_relative_calculation.py (hunk 2 above)
Comment (hannahbrucemacdonald) [reply perses/app/setup_relative_calculation.py]: Yes - I don't think saving no atom coordinates should be the default

**Comment by jchodera** [comment]
Is this PR ready to merge?

**Comment by hannahbrucemacdonald** [comment]
Yes!

**Pull request merged**